satellite/repair/repairer/segments.go

```diff
@@ -159,31 +159,31 @@ func (repairer *SegmentRepairer) Repair(ctx context.Context, path storj.Path) (e
 	}
 	newNodes, err := repairer.cache.FindStorageNodes(ctx, request)
 	if err != nil {
-		return Error.Wrap(err)
+		return false, Error.Wrap(err)
 	}
 
 	// Create the order limits for the PUT_REPAIR action
 	putLimits, putPrivateKey, err := repairer.orders.CreatePutRepairOrderLimits(ctx, bucketID, pointer, getOrderLimits, newNodes)
 	if err != nil {
-		return Error.Wrap(err)
+		return false, Error.Wrap(err)
 	}
 
 	// Download the segment using just the healthy pieces
 	rr, err := repairer.ec.Get(ctx, getOrderLimits, getPrivateKey, redundancy, pointer.GetSegmentSize())
 	if err != nil {
-		return Error.Wrap(err)
+		return true, Error.Wrap(err)
```

Comment:
Same question about why delete in this case?

Reply:
I should figure out how to check for database connectivity errors.  All of the other logic in Get is input validation or that the pointer has been deleted.

Reply:
Will add a comment to code, clarifying.  From what I can tell, .Get() only sets up the connection and can only fail from input validation.

Reply:
:+1: